pynocular/database_model.py

```diff
@@ -229,6 +288,7 @@ async def get(cls, *args: Any, **kwargs: Any) -> "DatabaseModel":
                 combination of parameters is invalid.
 
         """
+        resolve_references = kwargs.pop("resolve_references", False)
```

Comment:
What do we do if one of their columns is "resolve_references"?

I think you can also pull it out explicitly in the interface so it seems less special: `def get(cls, *args, resolve_references=False, **kwargs)`

Reply:
oh interesting. I tried that signature and it didnt work however I put the resolve_references after the `**kwargs`. Im not sure how to solve the first problem you mentioned. One way to solve it is to make a separate `get` function that will do the get with references. That is proably the safest. Thoughts?